lms/models/application_instance.py

```diff
@@ -18,6 +19,16 @@
 class ApplicationInstance(BASE):
     """Class to represent a single lms install."""
 
+    class Product(str, Enum):
+        BLACKBOARD = "BlackboardLearn"
+        CANVAS = "canvas"
+        MOODLE = "moodle"
+        D2L = "desire2learn"
+        BLACKBAUD = "BlackbaudK12"
+        ILIAS = "ilias"
+        SAKAI = "sakai"
+        UNKNOWN = "unknown"
```

Comment:
All values from the DB

Reply:
Can we remove Ilias? I have no idea what that is! And there's only one instance of it in the DB.

There's also "schoology" (seven instances in the DB). We may want to add that